extensions/shared/premium-blocks/render-premium-icon.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import PaidSymbol from './paid-symbol';
+
+const renderPremiumIcon = icon => {
+	if ( icon.src ) {
```

Comment:
should be consistent with the next `if` here, checking if the icon is defined before to access to the src property?

```suggestion
	if ( icon?.src ) {
```